charts/scalar-admin-k8s/templates/scalar-admin-k8s/job.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if not .Values.scalarAdminK8s.cronJob.enabled }}
```

Comment:
I think it is okay to create a job immediately when doing helm install. In other words, we can create a Job regardless of whether CronJob is enabled or not.

Reply:
Thank you for your review!

As we discussed in another meeting, I want to avoid running `Job` automatically when users deploy `CronJob` because the `Job` actually pauses (stops) ScalarDB/ScalarDL (i.e., pauses the user's system).

However, I agree that we should provide a way to test `Job` when users deploy `CronJob`.

So, instead of deploying `Job` automatically, I added the `NOTES.txt` to show `how to test the Job resource that will be created by CronJob` as follows. in ae450570c000bcda0fad217515e3763bc545659a .

* Job
  ```console
  $ helm install foo ~/github.com/scalar-labs/helm-charts/charts/scalar-admin-for-kubernetes/ -f ./foo.yaml --set scalarAdminForKubernetes.jobType=job
  NAME: foo
  LAST DEPLOYED: Fri Dec 29 15:23:55 2023
  NAMESPACE: default
  STATUS: deployed
  REVISION: 1
  TEST SUITE: None
  NOTES:
  CHART NAME: scalar-admin-for-kubernetes
  CHART VERSION: 1.0.0-SNAPSHOT
  APP VERSION: 1.2.0
  RELEASE NAME: foo
  
  The Job resource foo-scalar-admin-for-kubernetes that runs Scalar Admin for Kubernetes is created.
  
  You can see the "pause duration" as a command result that is output into the stdout of a container by running the following command:
  
    kubectl logs -l app.kubernetes.io/app=scalar-admin-for-kubernetes,app.kubernetes.io/instance=foo -n default
  ```
* CronJob
  ```console
  $ helm install bar ~/github.com/scalar-labs/helm-charts/charts/scalar-admin-for-kubernetes/ -f ./bar.yaml --set scalarAdminForKubernetes.jobType=cronjob
  NAME: bar
  LAST DEPLOYED: Fri Dec 29 15:24:15 2023
  NAMESPACE: default
  STATUS: deployed
  REVISION: 1
  TEST SUITE: None
  NOTES:
  CHART NAME: scalar-admin-for-kubernetes
  CHART VERSION: 1.0.0-SNAPSHOT
  APP VERSION: 1.2.0
  RELEASE NAME: bar
  
  The CronJob resource bar-scalar-admin-for-kubernetes that runs Scalar Admin for Kubernetes periodically is created.
  
  If you want to test the Job definition that will created by this CronJob, you can run the actual Job by running the following command:
  
    kubectl create job "bar-test-$(date +%s)" --from cronjob/bar-scalar-admin-for-kubernetes -n default
  
    NOTE: This command creates an actual Job and runs the Scalar Admin for Kubernetes. It pauses your ScalarDB or ScalarDL. In other words, it pauses transactions in your system.
  
  You can see the "pause duration" as a command result that is output into the stdout of a container by running the following command:
  
    kubectl logs -l app.kubernetes.io/app=scalar-admin-for-kubernetes,app.kubernetes.io/instance=bar -n default
  ```